app/api/init_api.py

```diff
@@ -64,6 +67,57 @@ def openapi(self) -> dict[str, Any]:
         return self.openapi_schema
 
 
+@asynccontextmanager
+async def lifespan(asgi_app: BanchoAPI) -> AsyncIterator[Never]:
```

Comment:
should this be AsyncIterator[None]?

Reply:
i remember trying that and it failing mypy